### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'slice') at ./node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js (streamwritable.js:57:1)

### DISCUSSION THREAD

**Discussion by hubsmoke** [Web3Auth/Web3Auth#474] [Help]
I keep getting error streamwritable.js:57 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'slice')
at ./node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js (streamwritable.js:57:1)

Whenever I initiate the modal this error comes up and the modal is stuck in a spinner and that's my entire experience using import { Web3Auth } from "@web3auth/web3auth";

I've had no issues integrating Torus in the past. Perhaps it's best to revert to an old version of Torus... however I'm trying to follow the docs and this error keeps coming up. Integrating into a React app on Goerli testnet

**Comment by himanshuchawla009** [comment]
can you paster your code snippet here?

**Comment by yashovardhan** [comment]
Hey @hubsmoke is this issue resolved?

**Reply by shahbaz17** [reply]
Marking this as done because of no response.
